hack/make.sh

```diff
@@ -48,16 +48,19 @@ function main ()
 	pouchd > $TMP/log 2>&1 &
 
 	# wait until pouch daemon is ready
+	i=0
 	while true;
 	do
-		COUNT=`ps -ef | grep pouchd | grep -v grep | wc -l`
-		if [ $COUNT = 0 ];then
-			echo "failed to start pouch daemon."
-			return 1
-		elif [ -S /var/run/pouchd.sock ];then
+		if [ -S /var/run/pouchd.sock ];then
+			echo "Succeed to start pouch daemon"
 			break
+		elif [ $i -gt 30 ];then
+			echo "Failed to start pouch daemon"
+			return 1
 		else
 			sleep 1
+			i=`expr $i + 1`
+			continue
 		fi
```

Comment:
This could be simplified as following:
```

else
		sleep 1
         	if (( $(( timeout--)) == 0 )) ; then
                    echo "Failed to start pouch daemon"
		     return 1
                fi
fi


```

Reply:
Timeout is a linux command name. It is not recommended to use timeout as variable name.
Maybe "if (( $(( daemon_timeout_time--)) == 0 )) ; then" can be used.

#timeout --help
Usage: timeout [OPTION] DURATION COMMAND [ARG]...
  or:  timeout [OPTION]
Start COMMAND, and kill it if still running after DURATION.

@Letty5411 What do you think?